solana/verify

```diff
@@ -74,5 +74,7 @@ if [ "$hash1" == "$hash2" ]; then
   exit 0;
 else
   printf "\033[0;31mFailed to verify\033[0m\n" >&2;
+  echo "JSON: $account_json" >&2;
```

Comment:
do we really want to echo the whole json and dump?

Reply:
it just prints the file name (so you can see it, and open if you want)